content/en/docs/tasks/debug-application-cluster/debug-running-pod.md

```diff
@@ -134,10 +134,11 @@ here because `kubectl run` does not enable [process namespace sharing](
 creates.
 
 {{< note >}}
-The `--target` parameter must be supported by the {{< glossary_tooltip
-text="Container Runtime" term_id="container-runtime" >}}. When not supported,
-the Ephemeral Container may not be started, or it may be started with an
-isolated process namespace.
+The `--target` parameter requires {{< glossary_tooltip text="kubelet"
+term_id="kubelet" >}} version `v1.18` or later and must be supported by the
+{{< glossary_tooltip text="Container Runtime" term_id="container-runtime" >}}.
+If these requirements are not met the Ephemeral Container may not start, or
+it may start with an isolated process namespace.
```

Comment:
The requirement for kubelet >= v1.18 is already implied by https://kubernetes.io/docs/setup/release/version-skew-policy/

@verb, do we need to document this explicitly; also, is it enough to hyperlink to the policy on supported versions?

A similar change to the docs for v1.19 specifically might be more relevant maybe?

Reply:
Hi Tim,

I suggested the change because of the exchange [here](https://github.com/kubernetes/enhancements/issues/277#issuecomment-737764301)

The thing I'm trying to clarify more explicitly is that an ephemeral container may be created, but it won't behave as expected in the following scenarios:

* kubelet versions 1.16 & 1.17, when the ephemeral container will be created but `--target` won't have any effect
* kubelet version >= 1.18 with a runtime that doesn't support targeting, for example crio.

I thought it might be better to be explicit, but I don't feel strongly about it.

Reply:
:wave: @verb .
It seems like it is important to indicate the version requirement?

What do you think about adding a version check after the prerequisite heading?
Also add a version field in the page's front matter:
```
min-kubernetes-server-version: 1.18
```
And a version check in the prerequisites section:

```
{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}

```

Similar to the topology management task page:
https://kubernetes.io/docs/tasks/administer-cluster/topology-manager/#before-you-begin